root/layout/components/sidebar/WorkSidebar.js

```diff
@@ -77,6 +81,26 @@ const WorkSidebar = ({$c, work}: Props): React.Element<'div'> => {
               </SidebarProperty>
             ) : null}
 
+            {length == null ? null : (
+              <SidebarProperty
+                className="work-length"
+                label={addColonText(l('Median Length'))}
```

Comment:
This should probably not be under “Work information” as this is not an information stored in the database. I would probably suggest to move it either under “Recordings” in the main panel or under a separate section in the sidebar.

“Median Length” is not about the work, it is about its recordings. If the section title refers to recordings, it would probably make more sense.

Reply:
The recording length is under "Recording information" on the recording sidebar even though it is almost always just derived from the tracks. IMO this does belong on the sidebar for sure, but if you think we should have a separate section under "Work information", then you can propose some ideas :) I do think the information "the median duration of this work, as far as we know, is this" is about the work, though...